src/styles/annotator/components/_index.scss

```diff
@@ -0,0 +1,12 @@
+@use 'tailwindcss/components';
```

Comment:
All `@use` directives must come before anything else in the module. In this case as in some others in our SASS modules, we want to enable the tailwind components layer _before_ importing/`@use`-ing the following components modules. We cannot use `@tailwind` here because it comes before other `@use`s.

The source of `tailwindcss/components` does effectively what we'd have to do to work around the `@use` limitation here. We could roll our own version of that wrapper if we felt we wanted to.